es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number} ${street}`;
```

Comment:
This isn't an exact conversion:

---

![image](https://user-images.githubusercontent.com/16143771/127084068-d8543165-bdaa-4ef4-b8e9-b53dd4ad67dd.png)

---

![image](https://user-images.githubusercontent.com/16143771/127084090-1028ca03-532d-43bf-b17d-3feb0d08ec59.png)